<ui version="4.0" >
 <class>Pref::Ui::SubtitleWidget</class>
 <widget class="QWidget" name="Pref::Ui::SubtitleWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>353</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>일반</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="auto_load_group" >
         <property name="title" >
          <string>자동으로 불러오기</string>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>파일 재생시에 다음에 해당하는 자막을 자동으로 불러옵니다.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="auto_load_combo" >
              <property name="sizeAdjustPolicy" >
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <item>
               <property name="text" >
                <string>자막을 자동으로 불러오지 않음</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>파일과 같은 이름의 자막</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>파일 이름을 포함하는 모든 자막</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>파일과 같은 경로에 있는 모든 자막</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>불러온 자막 중 다음에 해당하는 자막을 자동으로 선택합니다.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="auto_select_combo" >
              <property name="sizeAdjustPolicy" >
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <item>
               <property name="text" >
                <string>가장 첫번째 자막</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>파일과 같은 이름의 자막</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>불러온 모든 자막</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>모든 다른 언어</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>표시 위치</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>재생 시작시의 자막 표시 위치</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="init_pos_spin" >
              <property name="maximum" >
               <number>100</number>
              </property>
              <property name="value" >
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider" >
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="value" >
             <number>100</number>
            </property>
            <property name="tracking" >
             <bool>true</bool>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_13" >
              <property name="text" >
               <string>화면 위쪽</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_14" >
              <property name="text" >
               <string>화면 아래쪽</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="display_on_margin_check" >
         <property name="text" >
          <string>전체화면시 상하단 여백에도 자막을 표시하기</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>인코딩: </string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="encoding_combo" >
               <property name="editable" >
                <bool>true</bool>
               </property>
               <property name="sizeAdjustPolicy" >
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>353</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>글꼴</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>글꼴 선택</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QFontComboBox" name="font_family_combo" />
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>기본 크기</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QDoubleSpinBox" name="font_size_spin" >
             <property name="accelerated" >
              <bool>true</bool>
             </property>
             <property name="decimals" >
              <number>1</number>
             </property>
             <property name="value" >
              <double>5.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>크기 맞춤</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QComboBox" name="auto_scale_combo" >
             <item>
              <property name="text" >
               <string>끄기</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>화면 세로 길이에 맞추기</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>화면 가로 길이에 맞추기</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>화면 대각선 길이에 맞추기</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>55</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>353</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>다중/통합자막</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_24" >
         <property name="text" >
          <string>통합 자막을 다음 클래스 순서대로 표시합니다. 클래스들은 ,(쉼표)로 구분합니다. 지정되지 않은 클래스는 알파벳 순으로 표시합니다.
입력예: ENCC,EGCC,KRCC,KNCC</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="priority_edit" >
         <property name="text" >
          <string>ENCC,EGCC,KRCC</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>init_pos_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>356</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel" >
     <x>299</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>init_pos_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>273</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel" >
     <x>343</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
